mono/mini/method-to-ir.c

```diff
@@ -377,11 +377,20 @@ break_on_unverified (void)
 		G_BREAKPOINT ();
 }
 
+static void
+clear_error (MonoError *error)
+{
+	mono_error_cleanup (error);
+// Hijack mono_error_cleanup function to be local macro that also initializes for reuse.
+#define mono_error_cleanup(error) clear_error (error)
+	error_init_reuse (error);
+}
+
 static void
 clear_cfg_error (MonoCompile *cfg)
 {
-	mono_error_cleanup (&cfg->error);
-	error_init (&cfg->error);
+	clear_error (&cfg->error);
```

Comment:
Why the double `clear_error`?

Reply:
I don't understand. There is no double clear.

Reply:
There is one here and the exact same call at the following line.